internal/proto/data_coord.proto

```diff
@@ -688,4 +690,9 @@ message GcConfirmRequest {
 message GcConfirmResponse {
   common.Status status = 1;
   bool gc_finished = 2;
-}
\ No newline at end of file
+}
+
+message ReportDataNodeTtMsgsRequest {
+  common.MsgBase base = 1;
```

Comment:
Add a corresponding MsgType?

Reply:
Currently I use MsgType_DataNodeTt. ReportDataNodeTtMsgsRequest is a simple wrapper of DataNodeTt array. How about I add it in another PR? Put it to master first and pick to 2.2?